scss/_tags.scss

```diff
@@ -69,6 +69,7 @@
       height: 100%;
       content: "";
       background-color: currentcolor;
+      -webkit-mask-image: escape-svg(var(--#{$boosted-prefix}close-icon)); //stylelint-disable-line property-no-vendor-prefix
```

Comment:
I'm really wondering why we need to add this. Autoprefixer should do it automatically since it does it already for other definitions of `mask-image` in our project. Wondering if it has something to do with CSS vars.
Let's keep this modification but I'm gonna try to find why it does that.

Reply:
If the content is anything else than `mask-image: var(--o-close-icon);` the transformation done by Autoprefixer is OK... 🤔 
As soon as the CSS var is called `--o-*` it doesn't work 🤔

Reply:
Oh I think I know where it comes from 💡 
In Autoprefixer we have this kind of code `mistakes: ['-khtml-', '-ms-', '-o-']`. Our Boosted prefix si `o-`. There must be a conflict. If I change our prefix to `$boosted-variable-prefix:     ms-` it doesn't work neither.
So I think we can keep this modification but you should add a comment in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1553 that we could remove this line when the `$boosted-prefix` will be removed.

Reply:
See https://github.com/postcss/autoprefixer/issues/1484